tools/scripts/README.md

```diff
@@ -31,14 +39,14 @@ be registered in the determined repo. Here is the process:
    value to the artifact produced by CI, including the task environments and
    the agent AMIs.
 
-5. (optional) Run `./refresh-ubuntu-amis.py bumpenvs.yaml`.  This will fetch
-   the up-to-date Ubuntu AMIs for each region for each of the `*_master_ami`
-   and `*_bastion_ami` image tags in bumpenvs.yaml.  This isn't strictly
-   necessary; we just need to run it periodically, and now is a fine time. The
-   base images in environments/cloud/environments-packer.json also need to be
-   manually updated periodically. AWS GovCloud images need to be updated
-   manually as they are (at the time of this writing) missing or not current
-   in Ubuntu's AMI locator.
+5. Run `./refresh-ubuntu-amis.py --bumpenvs-yaml bumpenvs.yaml`.  This will
+   fetch the up-to-date Ubuntu AMIs for each region for each of the
+   `*_master_ami` and `*_bastion_ami` image tags in bumpenvs.yaml.  This isn't
+   strictly necessary; we just need to run it periodically, and now is a fine
+   time. The base images in environments/cloud/environments-packer.json also
+   need to be manually updated periodically. AWS GovCloud images need to be
```

Comment:
Nit: refresh-ubuntu-amis.py will now do it, they just have to go push those changes too.

Reply:
Actually there's another circular dependency here: we refresh the base AMIs after rebuilding the AMIs for the upcoming release... I volunteer to fix this if I merge environments back into Determined next week, FWIW.